openfisca_us/parameters/usda/disabled_programs.yaml

```diff
@@ -0,0 +1,14 @@
+description: Program participation that qualify individuals as disabled under USDA rules
+
+values:
+  2021-01-01:
+  - is_ssi_disabled
+  - ssdi
+  - permanently_disabled_veteran
+  - surviving_spouse_of_disabled_veteran
+  - surviving_child_of_disabled_veteran
+
+metadata: 
+  reference:
+  - title: 'SNAP Special Rules for the Elderly or Disabled'
+    href: https://www.fns.usda.gov/snap/eligibility/elderly-disabled-special-rules#Who%20is%20elderly?
```

Comment:
```suggestion
    href: https://www.fns.usda.gov/snap/eligibility/elderly-disabled-special-rules#Who%20is%20disabled?
```

Reply:
these point to the same page, all the elderly and disabled rules are lumped together